go/datas/remote_database_handlers.go

```diff
@@ -380,14 +380,14 @@ func handleRootPost(w http.ResponseWriter, req *http.Request, ps URLParams, cs c
 		d.Panic("Can't set Root to a non-present Chunk")
 	}
 
-	// Ensure that proposed new Root is a Map and, if it has anything in it, that it's <String, <Ref<Commit>>
+	// Ensure that proposed new Root is a Map and, if it has anything in it, that it's <String, <Ref<Value>>
 	v := types.DecodeValue(c, nil)
 	if v.Type().Kind() != types.MapKind {
 		d.Panic("Root of a Database must be a Map")
 	}
 	m := v.(types.Map)
-	if !m.Empty() && !isMapOfStringToRefOfCommit(m) {
-		d.Panic("Root of a Database must be a Map<String, Ref<Commit>>, not %s", m.Type().Describe())
+	if !m.Empty() && !isMapOfStringToRefOfValue(m) {
```

Comment:
Agree with raf. I think the server should protect itself against clients replacing map values with garbage.